Capstone_EDA.ipynb

```diff
@@ -58,7 +58,7 @@
     "\n",
```

Comment:
<p>Die Funktion übernehme ich auch in die Python Datei!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>